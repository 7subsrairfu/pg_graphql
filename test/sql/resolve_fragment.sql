select gql.dispatch($$
    {
      allBlogs(first: 1) {
        edges {
          cursor
          node {
            ...BaseBlog
            ownerId
          }
        }
      }
    }

    fragment BaseBlog on Blog {
      name
      description
    }
$$);
