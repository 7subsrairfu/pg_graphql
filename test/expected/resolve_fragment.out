select gql.dispatch($$
    {
      allBlogs(first: 1) {
        edges {
          cursor
          node {
            ...BaseBlog
            ownerId
          }
        }
      }
    }

    fragment BaseBlog on Blog {
      name
      description
    }
$$);
                                                                         dispatch                                                                         
----------------------------------------------------------------------------------------------------------------------------------------------------------
 {"data": {"allBlogs": {"edges": [{"node": {"name": "A: Blog 1", "ownerId": 1, "description": "a desc1"}, "cursor": "WyJibG9nIiwgMV0="}]}}, "errors": []}
(1 row)

